libr/debug/p/native/linux/reg/linux-x64.h

```diff
@@ -144,28 +144,36 @@ return strdup (
 "fpu	st6	.64	128	0\n"
 "fpu	st7	.64	144	0\n"
 
-"fpu	xmm0h	.64	160	0\n"
-"fpu	xmm0l	.64	168	0\n"
+"fpu    xmm0    .64 160 16\n"
```

Comment:
Can u update the other x64 profiles. Too? Not just the linux one

Reply:
Which ones? Windows does its own thing for example